Ginger/Ginger/AutomatePageLib/AddActionMenu/WindowExplorer/WindowExplorerCommon.cs

```diff
@@ -71,6 +74,10 @@ public static ITreeViewItem GetTreeViewItemForElementInfo(Amdocs.Ginger.Common.U
             {
                 TVI = HTMLElementInfoConverter.GetHTMLElementTreeItem(((HTMLElementInfo)EI));
             }
+            else if(EI.ElementObject is XmlNode)
+            {
+                TVI = AppiumElementInfoConverter.GetTreeViewItemFor(EI);
```

Comment:
it's common class but i see specific code for Appium...

Reply:
existing logic, I missed to discuss this with you last week (which I pinged and then forgot).